kubejs/server_scripts/enigmatica/kubejs/base/loot_tables/atum/chests/pyramid_chest.js

```diff
@@ -164,6 +164,21 @@ onEvent('chest.loot_tables', (event) => {
                     weight: 20,
                     count: [1, 2]
                 },
+                {
+                    item: Item.of('shrink:mob_bottle', '{entity:"alexsmobs:rattlesnake"}'),
+                    weight: 10,
+                    count: [1, 2]
```

Comment:
I'd rather leave these as guaranteed pairs of 2, rather than a chance. The aim of them is to provide a breeding pair. Same for Komodo and Kangaroo

Reply:
resolved, and updated the other bottles too